daemon/src/bitmex_price_feed.rs

```diff
@@ -212,4 +220,30 @@ mod tests {
         assert_eq!(quote.ask, Price::new(dec!(42641)).unwrap());
         assert_eq!(quote.timestamp.seconds(), 1632192000)
     }
+
+    #[test]
+    fn quote_from_now_is_not_old() {
+        let quote = dummy_quote_at(OffsetDateTime::now_utc());
+
+        let is_older = quote.is_older_than(1.minutes());
+
+        assert!(!is_older)
+    }
+
+    #[test]
+    fn quote_from_one_hour_ago_is_old() {
+        let quote = dummy_quote_at(OffsetDateTime::now_utc() - 1.hours());
+
+        let is_older = quote.is_older_than(1.minutes());
```

Comment:
nit: The test would be more meaningful if it would test the boundaries. E.g.:

```
let quote = dummy_quote_at(OffsetDateTime::now_utc() - 2.minutes());

let is_older = quote.is_older_than(1.minutes());
```

might also want to cover more boundaries.

Reply:
Well, it is really a nitpick isn't it :D

The test makes sure we are doing the comparison correctly, i.e. older and not more recent. The fact that it uses one minute is pretty obvious from the impl so risk is low, hence I didn't bother :)